plugins/processors/defaulter/defaulter.go

```diff
@@ -0,0 +1,62 @@
+package defaulter
+
+import (
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/processors"
+)
+
+const sampleConfig = `
+	## Ensure's a set of fields always exist on your metric(s) with at the specified default value.
+	## For any given field pair (key = default) under *[processors.defaulter.fields]*, if it's not set, or its value
+	## is 'empty' on the incoming metric, a new or updated metric is set on the metric with the specified default.
+	## 
+	#	[processors.defaulter.fields]
+	#	 field_1 = "bar"
+	#    time_idle = 0
+	#    is_error = true
```

Comment:
Follow this guide to match style with other plugins: https://github.com/influxdata/telegraf/wiki/SampleConfig

2 space indent, wrap line length, etc.